src/veh_interact.cpp

```diff
@@ -572,13 +589,13 @@ void veh_interact::do_install()
     }
     mvwprintz(w_mode, 0, 1, c_ltgray, _("Choose new part to install here:"));
     wrefresh (w_mode);
-    
+
```

Comment:
Uh this looks my fault, sorry about that. I actually have the C::B editor set up to strip trailing blanks and noticed it do so on occasion - not sure why it failed me here, as I was all over that file :/ Will keep an eye out for that in future.